src/Illuminate/Auth/CreatesUserProviders.php

```diff
@@ -16,32 +16,45 @@
     /**
      * Create the user provider implementation for the driver.
      *
-     * @param  string  $provider
+     * @param  string|null  $provider
      * @return \Illuminate\Contracts\Auth\UserProvider
      *
      * @throws \InvalidArgumentException
      */
-    public function createUserProvider($provider)
+    public function createUserProvider($provider = null)
     {
+        $provider = $provider ?: $this->getDefaultUserProvider();
+
         $config = $this->app['config']['auth.providers.'.$provider];
+        $driver = array_get($config, 'driver');
```

Comment:
Sure! Using Laravel's helper function makes this component (`Auth`) dependant on the framework and can't be used as a standalone component. It's a common practise across the framework's codebase.